appdata-fix.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/org.kde.kalk.appdata.xml b/org.kde.kalk.appdata.xml
+index cbd19dc..d62314d 100644
+--- a/org.kde.kalk.appdata.xml
++++ b/org.kde.kalk.appdata.xml
+@@ -1,6 +1,9 @@
+ <?xml version="1.0" encoding="utf-8"?>
+ <component type="desktop-application">
+   <id>org.kde.kalk</id>
++  <metadata_license>CC0-1.0</metadata_license>
++  <project_license>GPL-3.0-or-later</project_license>
++  <content_rating type="oars-1.1"/>
+   <name>Calculator</name>
+   <name xml:lang="ca">Calculadora</name>
+   <name xml:lang="ca-valencia">Calculadora</name>
+@@ -77,29 +80,30 @@
+   <developer_name xml:lang="x-test">xxKDE Communityxx</developer_name>
+   <metadata_license>CC0-1.0</metadata_license>
+   <description>
+-    <p>Kalk is a powerful calculator designed for mobile platform.</p>
```

Comment:
Not sure why you changed the description.

Reply:
I had an MR merged upstream for this, but basically the old description is grammatically incorrect and didn't convey that the application works on desktop as well.

Reply:
ok :)